config/jobs/image-pushing/k8s-staging-csi.yaml

```diff
@@ -0,0 +1,29 @@
+postsubmits:
+  # This is the github repo we'll build from. This block needs to be repeated
+  # for each repo.
```

Comment:
Eventually we'll probably want to have a generator script for all the various repos. There are too many to of them to create and maintain this file manually. But for now as a test it's okay.